README.md

```diff
@@ -113,24 +114,24 @@ const TrustlyWidgetBankSelected = (data) => {
 Trustly.selectBankWidget(establishData, TrustlyOptions, TrustlyWidgetBankSelected);
 ```
 
-Now you must to implement in your `webview` a way to handle with 3 events triggered by the `widget` and `lightbox` to handle with the bank selection, close or cancel action and when the authorization finished.
+Now, you must implement in your WebView a way to handle the 3 events triggered by the Widget and Lightbox to handle the bank selection, close or cancel action, and when the authorization ends.
```

Comment:
Slight grammatical update:

Now, implement a way in the WebView to handle the three events triggered by the Select Bank Widget and Lightbox: user selects a bank, user cancels the authorization, and the user completes the authorization.